src/accounts/views.py

```diff
@@ -47,3 +57,50 @@ class ServiceProviderLoginView(LoginView):
 # Login selection page view
 def login_selection(request):
     return render(request, "login_selection.html")
+
+
+def profile_view(request):
+    user = request.user
+    if user.user_type == "service_provider":
+        service_provider = ""
+        service_provider = get_object_or_404(service_provider, user=user)
+        # TODO: Service Provider
+        form = None
+
+        # If it's a POST request, we're updating the profile
+        # if request.method == "POST":
+        #     form = ServiceProviderForm(request.POST, instance=service_provider)
+        #     if form.is_valid():
+        #         form.save()
+        #         return redirect("profile_view")  # Redirect to avoid resubmission
+        # else:
+        #     form = ServiceProviderForm(instance=service_provider)
+
+        return render(
+            request,
+            "profile_service_provider.html",
```

Comment:
I'll be adding this as a follow up for #87. What is profile_service_provider.html?